server/journeys/screen/screen/controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { BaseJourneyController } from '../../base/controller'
+
+export class ScreenController extends BaseJourneyController {
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const saferCustodyOutcomeItems = await this.getReferenceDataOptionsForRadios(
+      req,
+      'outcome-type',
+      req.journeyData.saferCustodyScreening?.outcomeType || '',
```

Comment:
I think this needs to be res.locals.formResponses?.['saferCustodyOutcome'] || req.journeyData. saferCustodyScreening?.outcomeType?.code. That way it will use either what the user has entered or what's in session. The former is for when validation fails and the latter is when they either back through the journey or arrive from check your answers